exporter/signalfxexporter/README.md

```diff
@@ -144,9 +145,7 @@ will be replaced with a `_`.
   ```
 - `drop_histogram_buckets`:  (default = `false`) if set to true, histogram buckets will not be translated into datapoints with `_bucket` suffix but will be dropped instead, only datapoints with `_sum`, `_count`, `_min` (optional) and `_max` (optional) suffixes will be sent. Please note that this option does not apply to histograms sent in OTLP format with `send_otlp_histograms` enabled.
 - `send_otlp_histograms`: (default: `false`) if set to true, any histogram metrics receiver by the exporter will be sent to Splunk Observability backend in OTLP format without conversion to SignalFx format. This can only be enabled if the Splunk Observability environment (realm) has the new Histograms feature rolled out. Please note that histograms sent in OTLP format do not apply to the exporter configurations `include_metrics` and `exclude_metrics`.
-In addition, this exporter offers queued retry which is enabled by default.
-Information about queued retry configuration parameters can be found
-[here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md).
+In addition, this [exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md) offers queued retry which is enabled by default.
```

Comment:
```suggestion
In addition, this exporter offers [queued retry](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md) which is enabled by default.
```
The link here is to show documentation on queued retry options, not the exporter itself.

Reply:
I struggled a lot w/ this specific item -- ideally a link should describe what's on the other side of the link -- if the link was https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md#queued-retry, then [queued retry](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md#queued-retry) would be a proper link.

It's better to say "see the queued retry options in the [exporter documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)" or something like that...

Reply:
Your alternate suggestion works for me, I like that more than the original.

Just to try to clarify why I'm nitpicking, the wording of the original comment was "this exporter". The SignalFx exporter is an exporter which internally uses the exporter helper package for some reusable configuration options. The exporter helper is what's being linked to here. My thought is that "this exporter" in an exporter's documentation should refer to the exporter itself, not a widely used package that's used internally. Hopefully that makes sense.

To summarize, I'm mostly just against the wording of "this exporter" here when used for something other than the exporter for which the README is written. 👍